config/v1/zz_generated.crd-manifests/0000_10_config-operator_01_infrastructures-CustomNoUpgrade.crd.yaml

```diff
@@ -2226,6 +2247,26 @@ spec:
                           rule: 'self == oldSelf || (size(self) == 2 && isIP(self[0])
                             && isIP(self[1]) ? ip(self[0]).family() != ip(self[1]).family()
                             : true)'
+                      dnsRecordsType:
+                        allOf:
```

Comment:
Is this right? It looks weird, but this is what was generated when I moved the enum definition.

Reply:
D'oh, of course. Completely forgot that this is duplicated across all on-prem platforms. Fixing.